.github/workflows/tests.yml

```diff
@@ -17,9 +17,7 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: 16
-          # Note: we do not cache here because we're not running lerna
-          # bootstrap, so the ~/.npm dir won't have all dependencies, and if
-          # this job finishes first then we'll only cache root dependencies.
+          cache-dependency-path: '**/package-lock.json'
```

Comment:
Add a comment on why this isn't just the top-level package-lock?

Reply:
Actually it can just use the top-level one now. Updated.